pkg/workloads/cortex/onnx_serve/api.py

```diff
@@ -249,7 +266,17 @@ def start(args):
         sess = rt.InferenceSession(model_path)
         local_cache["sess"] = sess
         local_cache["input_metadata"] = sess.get_inputs()
+        logger.info(
```

Comment:
Should we only log these if debug == true?

Reply:
no i think it's useful to show